.github/contribution-guide.md

```diff
@@ -0,0 +1,10 @@
+### Thank You!
+
+I would like to Thank You, for considering contributing to this tiny module. I will be very grateful for your contribution.
+
+Important: Following these guidelines help to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+#### Pull Request
+
+- There is a pull request template, which needs to be filled for any pull request. It will be automatically created once you create a pull request, but you go through it here [PR-template](./PULL_REQUEST_TEMPLATE.md).
```

Comment:
```suggestion
- There is a pull request template that needs to be filled for any pull requests. It will automatically be generated once you create a pull request. Feel free to go through it [here](./PULL_REQUEST_TEMPLATE.md).
```